### PR TITLE
Capture Dataverse event and store it in azure storage table

### PR DESCRIPTION
**Pull request by Aymalla**
This PR is a spike to address how capture Dataverse event and store it in Azure storage table using Azure functions
- Http trigger azure function to act as webhook
- Durable azure function to save the events data to Azure storage table

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,44 @@
...
+![run-function](assets/run-function.png)
+
+### Test on Dataverse and Azure environment
+
+- Create new Azure function App with Java as runtime
+![create a new function app](assets/create-function-app.png)
+- Deploy function to the newly created function App
+![deploy to function app](assets/deploy-function-app.png)
+- Update storage account connection string Configuration > `AzureWebJobsStorage`
+![Update-storage-account-connection](assets/Update-sa-connection.png)
```
Comment (julioalex-rezende) [inline README.md]: this picture is not loading, probably to a typo on the name (uppercase letter)

File: src/main/java/dataverse/DataverseWebhookDurable.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+ * durable function
+ * 2. Send request to statusQueryGetUri in `StartHelloCities` response to get
+ * the status of durable function
+ * For more instructions, please refer https://aka.ms/durable-function-java
+ * 
+ * Please add com.microsoft:durabletask-azure-functions to your project
+ * dependencies
+ * Please add `"extensions": { "durableTask": { "hubName": "JavaTestHub" }}` to
+ * your host.json
+ */
```
Comment (julioalex-rezende) [inline src/main/java/dataverse/DataverseWebhookDurable.java]: out of date comments :)

**Review by julioalex-rezende — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Aymalla) [reply README.md]: Done

File: src/main/java/dataverse/DataverseWebhookDurable.java (hunk 2 above)
Comment (Aymalla) [reply src/main/java/dataverse/DataverseWebhookDurable.java]: resolved

**Pull request merged**